libdnf/repo/Repo.cpp

```diff
@@ -816,9 +817,7 @@ std::vector<Key> Repo::Impl::retrieve(const std::string & url)
     char tmpKeyFile[] = "/tmp/repokey.XXXXXX";
     auto fd = mkstemp(tmpKeyFile);
     if (fd == -1) {
-        char buf[1024];
-        (void)strerror_r(errno, buf, sizeof(buf));
-        auto msg = tfm::format("%s: mkstemp(): %s", __func__, buf);
+        auto msg = tfm::format("%s: mkstemp(): %s", __func__, std::system_category().message(errno));
```

Comment:
I'd drop the `__func__` here, I don't find it useful.

OTOH, the actual filename being created is missing and `mkstemp(): ` is not really an error message at all. What about something like:
```
auto msg = tfm::format("Error creating temporary file \"%s\": %s", tmpKeyFile, std::system_category().message(errno));
```

Reply:
OK. I'm going to change it.